README.md

```diff
@@ -46,21 +42,23 @@ or with a CDN:
-| [`initialize`](#initialize)        	| Method for identifying users and setting a JWT                                                                            	|
-| [`track`](#track)               	| Track custom events                                                                                                       	|
-| [`trackInAppClick`](#trackInAppClick)     	| Track when a user clicks on a button or link within a message                                                             	|
-| [`trackInAppClose`](#trackInAppClose)     	| Track when an in-app message is closed                                                                                    	|
-| [`trackInAppConsume`](#trackInAppConsume)   	| Track when a message has been consumed. Deletes the in-app message from the server so it won't be returned anymore        	|
-| [`trackInAppDelivery`](#trackInAppDelivery)  	| Track when a message has been delivered to a user's device                                                                	|
-| [`trackInAppOpen`](#trackInAppOpen)      	| Track when a message is opened and marks it as read                                                                       	|
-| [`trackPurchase`](#trackPurchase)       	| Track purchase events                                                                                                     	|
-| [`updateCart`](#updateCart)          	| Update _shoppingCartItems_ field on user profile                                                                          	|
-| [`updateSubscriptions`](#updateSubscriptions) 	| Updates user's subscriptions                                                                                              	|
-| [`updateUser`](#updateUser)          	| Change data on a user's profile or create a user if none exists                                                           	|
-| [`updateUserEmail`](#updateUserEmail)     	| Change a user's email and reauthenticate user with the new email address (in other words, we will call `setEmail` for you)                                                                                             	|
+| Method Name                                            | Description                                                                                                                         |
+| ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------- |
+| [`getInAppMessages`](#getInAppMessages)                | Either return in-app messages as a Promise or automatically paint them to the DOM if the second argument is passed `DisplayOptions` |
+| [`initialize`](#initialize)                            | Method for identifying users and setting a JWT                                                                                      |
```

Comment:
@pauljung14 Isn't `setEmail` for identifying the user?

Reply:
That's correct. So the flow goes like this: `setEmail` is part of the return signature for `initialize`. Once you initialize with an auth token an a callback to generate your JWT, then you can call setEmail to identify your user.

Reply:
Got it. Wondering if it's confusing to describe this method as being used to identify the user, when it's really a follow-on call?

Reply:
Updating